### PR TITLE
uncomment file index.js

### PR DESCRIPTION
**Pull request by tuanbui0509**
check please

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -7,15 +7,15 @@ var tom =  new Cat('Tom');
...
-// tom.die();
-// console.log(tom);
-// console.log(butch);
-// console.log("After being ate tom and jerry are died");
-// console.log(jerry);
-// console.log(tom);
\ No newline at end of file
+console.log("Before");
+console.log(jerry);
+console.log(tom);
```
Comment (buingoctuan99) [inline index.js:12]: Please add console.log(butch);

**Review by buingoctuan99 — COMMENTED** [review]
add console.log

File: index.js (hunk 1 above)
Comment (tuanbui0509) [reply index.js:12]: ### ### ### **s_sasasssss_**

Comment (buingoctuan99) [reply index.js:12]: okie

File: index.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ butch.sayHi();
 console.log("Before");
 console.log(jerry);
 console.log(tom);
+console.log(butch);
```
Comment (tuanbui0509) [inline index.js:13]: okie this exaclly

**Pull request merged**